plugins/katello/developers.md

```diff
@@ -127,105 +127,40 @@ Prior to branching the Katello repositories, the release nanny should figure out
 
 ### Step 3: Setup Your Environment
 
-There are a number of repositories that will need to be tagged and branched throughout the process. We recommend organizing this before hand so that you have a common place to reference and work through these tasks within throughout the release process. Clone the following:
-
-```
-git clone https://github.com/Katello/katello
-git clone https://github.com/Katello/katello-installer.git
-git clone https://github.com/Katello/katello-misc
-git clone https://github.com/Katello/katello-agent
-git clone https://github.com/Katello/hammer-cli-katello
-```
-
-Note that some repositories contain multiple spec-files and thus packages:
+All you need is [tool_belt](https://github.com/ehelms/tool_belt).
 
+```sh
+git clone https://github.com/ehelms/tool_belt
+cd tool_belt
+bundle
+./tools.rb
 ```
-katello
-  rubygem-katello.spec
-  deploy/katello.spec
-
-katello-misc
-  certs-tools/katello-certs-tools.spec
-  repos/katello-repos.spec
-  utils/katello-utils.spec
 
-katello-agent
-  katello-agent.spec
+### Step 4: Configure the Release
 
-katello-installer
-  katello-installer.spec
+Create a 2.1 release configuration in tool_belt at `configs/katello_21.yaml`. You may copy the config from the previous release and update accordingly. Once configured, run setup:
```

Comment:
Mind updating the references throughout from 2.1 to 3.5 so anyone reading it looks at a more recent configuration?

Reply:
I think it's fine to use 2.0 -> 2.1 because at the top of the docs, it states that 2.0 is the currently released Katello, and 2.1 is the release pending. I don't find it confusing, and I really don't want to have to keep these version numbers current.